### PR TITLE
Add base Debug UI for easier debugging purposes

### PR DESCRIPTION
**Pull request by kubalakowski**
Press F12 to show/hide

### CODE REVIEW

File: UIElementsSchema/Unity.Cloud.Collaborate.Components.ChangeListEntries.xsd [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Brodie0) [inline UIElementsSchema/Unity.Cloud.Collaborate.Components.ChangeListEntries.xsd:1]: what these .Cloud files do?

**Comment by Brodie0** [comment]
Can you paste here a link to tutorial/codebase on which these changes are based? In case it would need some update or just for sharing knowledge?

**Review by Brodie0 — APPROVED** [review]

**Comment by kubalakowski** [comment]
> Can you paste here a link to tutorial/codebase on which these changes are based? In case it would need some update or just for sharing knowledge?

Based on Unity docs
> https://docs.unity3d.com/Manual/UIE-simple-ui-toolkit-workflow.html

**Comment by kubalakowski** [comment]
Also used these YT videos
https://www.youtube.com/watch?v=6zR3uvLVzc4&
https://www.youtube.com/watch?v=NQYHIH0BJbs

**Pull request merged**